.github/workflows/deploy.yml

```diff
@@ -174,6 +175,7 @@ jobs:
       run: |
         DEPLOY_ENV="ci${GITHUB_RUN_ID}"
         echo "DEPLOY_ENV=$DEPLOY_ENV" | tee -a $GITHUB_ENV
+        echo "RESOURCE_REMOVAL_POLICY=RETAIN" | tee -a $GITHUB_ENV
```

Comment:
Am I misunderstanding something, or does this make the deployment different from the actual production deployment? Wouldn't that make it likely that `test-prod-copy-upgrade` passes but `deploy` might fail?

Reply:
`RESOURCE_REMOVAL_POLICY=RETAIN` makes our upgrade testing deployment similar to prod. Until now, removal policy was set to `RETAIN` only for prod deployments.

Reply:
Where is it set to retain prod deployments? `git grep -i RETAIN` doesn't show anything prod-related AFAICT.